config/locales/errors/en.yml

```diff
@@ -52,7 +52,10 @@ en:
       inclusion: is not included in the list
       invalid_calling_area: Calls to that phone number are not supported. Please try
         SMS if you have an SMS-capable phone.
-      invalid_phone_number: Phone number is not valid
+      invalid_phone_number:
+        international: 'Enter a phone number with the correct number of digits. Example:
+          (201) 555-0123'
```

Comment:
Are these example numbers supposed to be dynamic based on the country selected? It's not entirely clear to me from the ticket, but this number format wouldn't be valid for every international number.

Reply:
Based on what is provided in the ticket, the example numbers are not dynamic. 

Yeah, the format is not universal to everyone. I think that the intent of this change was to give guidance on how to validate their phone numbers. This is definitely worth a discussion with the team--I can lead that from the engineering side.